crates/hir-ty/src/tests/macros.rs

```diff
@@ -194,15 +194,15 @@ fn expr_macro_def_expanded_in_various_places() {
             !0..6 '1isize': isize
             !0..6 '1isize': isize
             !0..6 '1isize': isize
-            39..442 '{     ...!(); }': ()
+            39..442 '{     ...!(); }': {unknown}
             73..94 'spam!(...am!())': {unknown}
             100..119 'for _ ...!() {}': fn into_iter<isize>(isize) -> <isize as IntoIterator>::IntoIter
-            100..119 'for _ ...!() {}': {unknown}
+            100..119 'for _ ...!() {}': <isize as IntoIterator>::IntoIter
```

Comment:
This test has again associated type normalization and `{unknown}` accuracy in fluff; we're used to that with the new solver, I'm not sure under what condition that happens.